pkg/databricks/client.go

```diff
@@ -52,7 +52,7 @@ func NewClient(ctx context.Context, httpClient *http.Client, hostname, accountHo
 	}
 
 	if accountHostname == "" {
-		accountHostname = "accounts." + defaultHost
+		accountHostname = "accounts." + hostname
```

Comment:
This should probably be set based on the examples I gave in my comment in auth.go.